### ISSUE TITLE
UX shows "not yet deployed" for a new local deployment which has been associated with a deployment on connect

### ISSUE THREAD

**Issue by dotNomad** [posit-dev/publisher#2349]
Reported as "weird" from Neil, during demo.

Can we adjust the wording to indicate that an update has not been deployed from here but that this project has been deployed?

**Comment by dotNomad** [comment]
We do update the wording when associating a Deployment:

![Image](https://github.com/user-attachments/assets/4b73e59f-7d57-46ca-a43e-a4bf2c0221ac)

We may need clarification on if the "Not Yet Deployed" wording is weird prior to the association or if the "Not Yet Updated" wording is weird.

**Comment by dotNomad** [comment]
@nealrichardson would love your thoughts on this one 😄

**Comment by nealrichardson** [comment]
I don't think "Not Yet Updated" is useful information in this context. It makes me think something is wrong. 

If you're saying "you or someone else has deployed this before, just not from this IDE on this machine", I'm not sure how much difference that makes to me, no matter how technically true that is.

If you're saying "you have unpublished changes", that would be true regardless of where I've published from before.

Why shouldn't this basically look the same as it does when I've previously published on this computer?

**Comment by dotNomad** [comment]
All fantastic points.

> Why shouldn't this basically look the same as it does when I've previously published on this computer?

Great question. Here is what is shown when a Deployment has been published successfully with Publisher:

![Image](https://github.com/user-attachments/assets/3557b9b4-816c-4557-b913-b3f95e237f96)

We get the timestamp from our Deployment record. We could use the same messaging and just not include the timestamp since it is not present in the Deployment record.

The "View Content" button is still around to have that "double check I pointed this to the right place" flow for those that want that, and the messaging becomes consistent to avoid the signaling of something different and possibly wrong.

**Comment by nealrichardson** [comment]
Could you get the "last deployment" timestamp from the Connect API? (Aside, and feel free to file this as YAGNI for now if you want: it's probably useful information to know whether the latest deployment on Connect matches the local deployment record, for other cases where someone else did the most recent deployment.)

Other aside: "Last Deployment Successful" feels redundant. I know you're distinguishing the case where the last (attempted) deployment failed, but all I want (as a user) from this space is to see the last time the content was updated on Connect.

**Comment by dotNomad** [comment]
> it's probably useful information to know whether the latest deployment on Connect matches the local deployment record, for other cases where someone else did the most recent deployment

Absolutely. We have set aside all potential work of syncing with the Connect server to keep the first release simple, but there are quite a few cases where we want to implement features exactly like this.

Thanks for the suggestion 😄
